### ISSUE TITLE
Translating Terminologies 

### ISSUE THREAD

**Issue by nootanghimire** [reactjs/ne.reactjs.org#2]
A lot of Nepali developer community are already familiar with the English terminology and sometimes it creates a lot of confusion if we translate them ( in a inconsistent way)

Examples like: "technology stack" is a fairly technical term, and we need to agree on if we want to transliterate the phrase, leave it in plain english or try and localize it to Nepali. 

Similarly, in the phrase "The view dispatches action", there's a lot of word there that gives a specialized meaning when put into this context, but the concept does not really exist in Nepali. We need to come up with a plan to handle these scenarios.

**Comment by samundrak** [comment]
I am making a new issue about all Glossary and will pin it.

**Comment by samundrak** [comment]
जहाँ सम्म मलाई लाग्छ तेस्तो टेक्निकल कुरा english मै राख्दिनु पर्छ, हामीले केवल बुझाउने प्रयास गर्नुपर्छ

**Comment by nootanghimire** [comment]
Closing this in favor of #3 !
